stl/inc/xstring

```diff
@@ -4180,6 +4185,12 @@ public:
         _Eos(static_cast<size_type>(_Result_size));
     }
 #pragma warning(pop)
+
+#if _HAS_CXX23
+    template <class _Operation>
+    constexpr void _Resize_and_overwrite(_CRT_GUARDOVERFLOW const size_type _New_size, _Operation _Op) {
+        return resize_and_overwrite(_New_size, _Op);

```

Comment:
Might be it worth to make `_Resize_and_overwrite` calling `resize_and_overwrite` instead of the opposite, and use `_Pass_fn` in `resize_and_overwrite` ?

Reply:
> Might be it worth to make `_Resize_and_overwrite` calling `resize_and_overwrite` instead of the opposite

That's what's currently happening - did you intend to say the other order?

I think it's desirable for the Standard name, when available, to not be a wrapper. I don't think we do that consistently elsewhere, but in this new case it seems reasonable. Thus I think the code is fine as-is.

Reply:
Yes I meant the opposite.

> I think it's desirable for the Standard name, when available, to not be a wrapper

Ok. I did it for another reason, specifically to avoid extra copying of a potentially heavy object, but let's keep it for this reason then.